pkg/kubelet/server/server.go

```diff
@@ -552,15 +542,36 @@ func (s *Server) InstallDebuggingDisabledHandlers() {
 	s.addMetricsBucketMatcher("containerLogs")
 	s.addMetricsBucketMatcher("runningpods")
 	s.addMetricsBucketMatcher("pprof")
-	s.addMetricsBucketMatcher("logs")
 	paths := []string{
 		"/run/", "/exec/", "/attach/", "/portForward/", "/containerLogs/",
-		"/runningpods/", pprofBasePath, logsPath}
+		"/runningpods/", pprofBasePath}
```

Comment:
by removing this from the InstallDebuggingDisabledHandlers method, we no longer get a helpful `Debug endpoints are disabled` message when requesting /logs on the readonly port, or when debug/log handling is disabled